### DISCUSSION TITLE
/is accept command. Removing arguments

### DISCUSSION THREAD

**Discussion by Evansyjord** [BG-Software-LLC/SuperiorSkyblock2#1366] [Implemented Feature Requests] [answered]
Hi, just a quick one. Lets say i invite a user, i execute the command "/is invite <username>" that said user would have to enter the command "/is accept <username/islandname>" would it be possible to remove the args after /is accept. I understand its probably implemented to ensure the user is joining the correct island but if we have it in the config as a bool option (True / False) then it gives the server owner the control to either add it or not if they dont like the feature. But lets say the server has this option under "island-names:" in the main config file:
island-names:
  required-for-creation: false
  ...
  ...
  ...
  is-accept-args: true/false
If set to TRUE then the command would be "/is accept <username/islandname> and would act like it does right now
BUT if the option is set to false then the command would just be "/is accept" and it would accept the invite of the last sent invite at the current time of command execution but the args would be totally optional in case the user was getting spammed by 50 different invites they could just specify that users name but once again, that would be completely optional

**Comment by Tptuaazn** [comment]
Without args will accept the last invitation.

**Comment by OmerBenGera** [comment, marked as answer]
Implemented in latest dev build
